index.slim

```diff
@@ -110,7 +110,7 @@ html lang="en"
             div.border
               h3.donation-title #{donation_title}
               p.progress Current donated amount:
-              p.progress#total-fund-raised.total-fund-raised.odometer 0.00
+              p.progress#total-fund-raised.total-fund-raised 0.00
```

Comment:
We can also remove corresponding javascript and stylesheet files to odometer if we want to remove animation.